plugins/trace_api_plugin/include/eosio/trace_api/common.hpp

```diff
@@ -63,6 +63,8 @@ namespace eosio::trace_api {
    using exception_with_context = std::tuple<const std::exception_ptr&, char const *, uint64_t, char const *>;
    using exception_handler = optional_delegate<void, const exception_with_context&>;
 
+   using log_handler = optional_delegate<void, const std::string&>;
```

Comment:
`optional_delegate` was moved to `fc` in `release/2.0.x`. Also it now supports syntax: `optional_delegate<void(const std::string&)>`

Reply:
I'm not sure how you have `optional_delegate` here without having a conflict on merge.

Reply:
yeah that was odd but its fixed now